### PR TITLE
Append some args to examples for #508

### PR DESCRIPTION
**Pull request by bakfoo**

### CODE REVIEW

**Comment by bakfoo** [comment]
I think these fails were caused by old "test_example.sh". You'd need to change the new one that I have created pull request on chainer-test (https://github.com/pfnet/chainer-test/pull/85 ).

**Comment by unnonouno** [comment]
Please check coding style.

```
pip install hacking
cd chainer
flake8
```

**Comment by bakfoo** [comment]
I'm sorry for caring nothing for PEP8. I keep it in mind from now.

File: examples/ptb/train_ptb.py [hunk 1]
```diff
@@ -30,14 +30,27 @@
...
+parser.add_argument('--epoch', '-e', default=39, type=int,
+                    help='number of epochs to learn')
+parser.add_argument('--unit', '-u', default=650, type=int,
+                    help='number of units')
+parser.add_argument('--batchsize', '-b', type=int, default=20,
+                    help='learning minibatch size')
+parser.add_argument('--bploplen', '-l', type=int, default=35,
+                    help='length of truncated BPTT')
+parser.add_argument('--gradclip', '-c', type=int, default=5,
+                    help='graddient norm threshold to clip')
```
Comment (unnonouno) [inline examples/ptb/train_ptb.py]: s/graddient/gradient/

File: examples/ptb/train_ptb.py [hunk 2]
```diff
@@ -30,14 +30,27 @@
                     help='Resume the optimization from snapshot')
 parser.add_argument('--gpu', '-g', default=-1, type=int,
                     help='GPU ID (negative value indicates CPU)')
+parser.add_argument('--epoch', '-e', default=39, type=int,
+                    help='number of epochs to learn')
+parser.add_argument('--unit', '-u', default=650, type=int,
+                    help='number of units')
+parser.add_argument('--batchsize', '-b', type=int, default=20,
+                    help='learning minibatch size')
+parser.add_argument('--bploplen', '-l', type=int, default=35,
```
Comment (unnonouno) [inline examples/ptb/train_ptb.py]: `bproplen` ?

File: examples/ptb/train_ptb.py [hunk 3]
```diff
@@ -30,14 +30,27 @@
...
 
-n_epoch = 39   # number of epochs
-n_units = 650  # number of units per layer
-batchsize = 20   # minibatch size
-bprop_len = 35   # length of truncated BPTT
-grad_clip = 5    # gradient norm threshold to clip
+n_epoch = args.epoch   # number of epochs
+n_units = args.unit  # number of units per layer
+batchsize = args.batchsize   # minibatch size
+bprop_len = args.bploplen   # length of truncated BPTT
```
Comment (unnonouno) [inline examples/ptb/train_ptb.py]: `bproplen`?

**Comment by bakfoo** [comment]
Very sorry about the typos...

**Comment by unnonouno** [comment]
Thank you very much! I checked this code in my environment.

**Pull request merged**